public/assets/css/productCard.css

```diff
@@ -0,0 +1,173 @@
+/* Product Card Container */
+.productCardContainer > .products {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+    margin: auto;
+    max-width: 1110px;
+    padding: 1rem;
+}
+
+@media screen and (max-width: 768px) {
+    .productCardContainer > .products {
+        padding: 0.25rem;
+    }
+}
+
+/* Product Card */
+.productCard {
+    background-color: var(--productCardBackgroundColor);
+    border-radius: 3px;
+    box-shadow: var(--shadowColor) 0 5px 5px;
+    display: flex;
+    margin: 1rem;
+    transition: ease-out 250ms;
+    flex-flow: column nowrap;
+    overflow: clip;
+    width: 20rem;
+    z-index: 1;
+}
+
+.productCard:hover {
+    box-shadow: var(--shadowColor) 0 8px 10px;
+    cursor: pointer;
+    transform: scale(1.025);
+    transition: ease-out 250ms;
```

Comment:
foster transition use inside element rather than hover or active effect, with this process the effect will trigger when you hover / when you leave the element.